tools/ci_build/github/azure-pipelines/linux-ci-pipeline.yml

```diff
@@ -54,7 +54,7 @@ jobs:
          python3 -m pip uninstall -y ort-nightly-gpu ort-nightly onnxruntime onnxruntime-gpu -qq
          cp $(Build.SourcesDirectory)/tools/ci_build/github/linux/docker/scripts/manylinux/requirements.txt $(Build.BinariesDirectory)/requirements.txt
          #Test ORT with the latest ONNX release.
-         sed -i 's/git+http:\/\/github\.com\/onnx\/onnx.*/onnx==1.9.0/' $(Build.BinariesDirectory)/requirements.txt
+         sed -i 's/git+http:\/\/github\.com\/onnx\/onnx.*/onnx==1.10.1/' $(Build.BinariesDirectory)/requirements.txt
```

Comment:
Does this PR has dependency on latest onnx version, and not supported by 1.9.0?

I did not see 1.10.1 supported in the following table:
https://github.com/microsoft/onnxruntime/blob/master/docs/Versioning.md#version-matrix

Reply:
Yes, my change does have a dependency on latest onnx 1.10.1 and the onnx submodule was set to 1.10.1 in the source tree when I did the last change hence the 1.10.1 version change here.

Reply:
done @tianleiwu thanks for the review